backend/src/StdLibExecution/Libs/String.fs

```diff
@@ -636,6 +638,26 @@ let fns : List<BuiltInFn> =
         | _ -> incorrectArgs ())
       sqlSpec = NotYetImplemented
       previewable = Pure
+      deprecated = NotDeprecated }
+
+    { name = fn "head" 0
```

Comment:
fwiw this could be a package fn, I think (toList, List.head) - or is it better this way for perf reasons or something?